src/main/java/net/sf/jabref/wizard/auximport/AuxSubGenerator.java

```diff
@@ -224,8 +222,8 @@ private boolean parseAuxFile(String filename) {
                             }
                         }
                     }
-                } // line != null
-                else {
+                } else {
+                    // line != null
                     weiter = false;
```

Comment:
I do not speak enough German to be fully sure, but that was my interpretation of it as well. :-) (Except that continue is a quite bad choice... ;-))